core/templates/pages/contributor-dashboard-admin-page/navbar/contributor-dashboard-admin-navbar.component.html

```diff
@@ -0,0 +1,148 @@
+                    <span *ngIf="profilePictureDataUrl">
+                      <!--
+                        Lighthouse flags <img> elements that don't have alt attributes:
+                        If the image acts as decoration and does not provide any useful content, give it an empty alt=""
+                        https://web.dev/image-alt/?utm_source=lighthouse&utm_medium=devtools#how-to-add-alternative-text-to-images
+                      -->
+                      <img [src]="profilePictureDataUrl"
+                           class="oppia-navbar-profile-picture rounded-circle"
+                           alt="">
+                    </span>
+                    <div *ngIf="!profilePictureDataUrl" class="dropdown-toggle">
+                      <i class="material-icons md-40 oppia-dropdown-toggle-icon">&#xE853;</i>
+                    </div>
+                  </div>
+                </a>
+                <ul ngbDropdownMenu class="dropdown-menu dropdown-menu oppia-navbar-dropdown"
+                    role="menu"
+                    (mouseenter)="activateProfileDropdown()"
+                    (mouseleave)="deactivateProfileDropdown()">
+                  <li>
+                    <a (click)="onMouseoutProfilePictureOrDropdown($event)"
+                       [href]="profileUrl"
+                       class="dropdown-item oppia-dropdown-item-text">
+                      <strong>{{ username }}</strong>
+                    </a>
+                  </li>
+                  <hr class="oppia-top-right-menu-item-separator">
+                  <li>
+                    <a (click)="onMouseoutProfilePictureOrDropdown($event)"
+                       href="/creator-dashboard"
+                       class="dropdown-item oppia-dropdown-item-text">
+                      Creator Dashboard
+                    </a>
+                  </li>
+                  <li>
+                    <a (click)="onMouseoutProfilePictureOrDropdown($event)"
+                       href="/notifications"
+                       class="dropdown-item oppia-dropdown-item-text">
+                      Notifications
+                    </a>
+                  </li>
+                  <li>
+                    <a (click)="onMouseoutProfilePictureOrDropdown($event)"
+                       href="/preferences"
+                       class="dropdown-item oppia-dropdown-item-text">
+                      Preferences
+                    </a>
+                  </li>
+                  <hr class="oppia-top-right-menu-item-separator">
+                  <li>
+                    <a (click)="onMouseoutProfilePictureOrDropdown($event)"
+                       [href]="logoutUrl"
+                       class="dropdown-item oppia-dropdown-item-text">
+                      Logout
+                    </a>
+                  </li>
+                </ul>
+              </li>
+            </ul>
```

Comment:
This is mostly the same as what is in core/templates/pages/admin-page/navbar/admin-navbar.component.html, right? Would it be possible to extract this into a component to not have to duplicate here?